### PR TITLE
Applies black formatting

### PR DESCRIPTION
**Pull request by DavidMinarsch**
## Proposed changes

This PR formats the codebase according to black.

## Fixes

-

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have checked that the documentation about the `aea cli` tool works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/list.py [hunk 1]
```diff
@@ -60,20 +83,20 @@ def _get_item_details(ctx, item_type) -> List[Dict]:
 def connections(ctx: Context):
     """List all the installed connections."""
     result = _get_item_details(ctx, "connection")
-    print(format_items(sorted(result, key=lambda k: k['name'])))
+    print(format_items(sorted(result, key=lambda k: k["name"])))
```
Comment (Totoual) [inline aea/cli/list.py]: Cannot use logger here over print ?

File: aea/cli/list.py [hunk 2]
```diff
@@ -60,20 +83,20 @@ def _get_item_details(ctx, item_type) -> List[Dict]:
...
 
 
 @list.command()
 @pass_ctx
 def protocols(ctx: Context):
     """List all the installed protocols."""
     result = _get_item_details(ctx, "protocol")
-    print(format_items(sorted(result, key=lambda k: k['name'])))
+    print(format_items(sorted(result, key=lambda k: k["name"])))
```
Comment (Totoual) [inline aea/cli/list.py]: same

File: aea/cli/list.py [hunk 3]
```diff
@@ -60,20 +83,20 @@ def _get_item_details(ctx, item_type) -> List[Dict]:
...
 
 
 @list.command()
 @pass_ctx
 def skills(ctx: Context):
     """List all the installed skills."""
     result = _get_item_details(ctx, "skill")
-    print(format_items(sorted(result, key=lambda k: k['name'])))
+    print(format_items(sorted(result, key=lambda k: k["name"])))
```
Comment (Totoual) [inline aea/cli/list.py]: same

File: aea/cli/list.py (hunk 1 above)
Comment (DavidMinarsch) [reply aea/cli/list.py]: We decided to use print because this is a large output and logger prepends too much

File: aea/cli/list.py (hunk 2 above)
Comment (DavidMinarsch) [reply aea/cli/list.py]: see above

**Pull request merged**